### PR TITLE
Add possibility to remove a favorite from favorites page

### PR DESCRIPTION
**Pull request by theblindspring**
This pull request addresses issue #969.

### CODE REVIEW

File: app/views/favorites/of_user.ctp [hunk 1]
```diff
@@ -52,7 +52,7 @@ $this->set('title_for_layout', $pages->formatTitle($title));
 </div>
 
 <div id="main_content">
-    <div class="module">
+    <div class="module" id="favorites-list" data-success="<?php echo __("Successfully Removed Favorite", true); ?>" >
```
Comment (alanfgh) [inline app/views/favorites/of_user.ctp]: The second and third words in the message string should not be capitalized.

**Comment by trang** [comment]
It's looking good :D Still a few fixes before I can merge this:

* The size of the icons should be bigger (24x24 instead of 16x16).
* The color of the icon should be grey (`#ccc`) instead of green, and black (`#000`) on hover.
* The string `Successfully removed favorite` should be `Favorite successfully removed.` (with the punctuation).

**Comment by alanfgh** [comment]
I agree: "Favorite successfully removed." is better.

**Comment by theblindspring** [comment]
Hey @trang,

What browser are you running on? My computer shows the icons as black, not green.

**Comment by trang** [comment]
Found the problem: if you're logged in you will see the icons are green. If you're not logged in they will be black.

Your icons are displayed all the time but they shouldn't. So there's one more thing to work on:

* The "remove" and "undo" icons should be displayed only for the user who owns the favorites. Other people shouldn't see these icons, since they shouldn't be able to delete favorites that don't belong to themselves.

**Comment by theblindspring** [comment]
Hmmm, I was logged in and they were black. I will work on getting them to only appear if you are logged in.

**Comment by trang** [comment]
Actually you're right, it's not a matter of being logged in, it's a matter of browser. They appear black on Chrome, green on Firefox.

**Pull request merged**